### ISSUE TITLE
Shallow imports break compatibility with Angular 9+

### ISSUE THREAD

**Issue by AgentEnder** [relair/odata-data-source#8]
Importing angular material modules like this is no longer supported.
```
import { MatSort, MatPaginator } from '@angular/material';
```

Instead, it needs to be like this:

```
import { MatPaginator } from '@angular/material/paginator';
import { MatSort } from '@angular/material/sort';
```

This is solved by PR #7

**Comment by devemanager** [comment]
```
 ERROR in node_modules/odata-data-source/lib/odata-data-source.d.ts:3:39 - error TS2306: File 'D:/LS-Graph-Toturial/node_modules/@angular/material/index.d.ts' is not a module.
    
    3 import { MatSort, MatPaginator } from '@angular/material';
```
I got an error also.

**Comment by AgentEnder** [comment]
Yeah, this is a simple case where the IVY engine has a breaking change with shallow imports. Really simple fix, have not heard anything from @relair yet. If you need to use this for now, I forked, updated, and published on npm as [@agentender/odata-data-source](https://www.npmjs.com/package/@agentender/odata-data-source)

**Comment by relair** [comment]
Those have nothing to do with IVY change, it's just that a new version of angular material have a breaking change which requires to make imports in different way.
When I find some time I will address it, as I will need to come up with some better versioning system, so it will be clear which version is supported, as making changes would make it incompatible with the older version of material. I will probably do what I did for my other library which will make major version be the angular version it supports.

**Comment by AgentEnder** [comment]
Closed by 7f1d0c9924f5e19e94a231923fc5523c0a46d191
